packages/common/src/ado_base/mod.rs

```diff
@@ -39,6 +39,9 @@ pub enum AndromedaMsg {
     UpdateAppContract {
         address: String,
     },
+    UpdateProcessContract {
```

Comment:
Is this required as a shared message type? This is only used in certain ADOs, right?